dlp/api/Snippets.Tests/DeidentifyWithReplacementTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class DeidentifyWithReplacementTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+
+        public DeidentifyWithReplacementTests(DlpTestFixture fixture) => _fixture = fixture;
+        [Fact]
+        public void TestDeidentifyWithReplace()
+        {
+            var text = "My name is Alicia Abernathy, and my email address is aabernathy@example.com and abc@gmail.com.";
+            var expected = "My name is Alicia Abernathy, and my email address is [email-address] and [email-address].";
+            var result = DeidentifyWithReplacement.Deidentify(_fixture.ProjectId, text);
+            Assert.NotEqual(text, result.Item.Value);
+            Assert.Contains("[email-address]", result.Item.Value);
+            Assert.Equal(expected, result.Item.Value);
```

Comment:
These are all redundant, you really only need:

```suggestion
            Assert.Equal(expected, result.Item.Value);
```

Reply:
Done.